### ISSUE TITLE
Is there a way to display all reactions from a model?

### ISSUE THREAD

**Issue by MauroBattipede** [zakandrewking/escher#385]
Hello everybody. I am using ESCHER on my jupyter nootebook. As somebody already asked: Is there a simple way to draw every reaction encoded within the .json and have it auto-spread? Or do I have do select each reaction manually via the widget?

Thanks

**Comment by ghost** [comment]
@MauroBattipede yes, For those who are using the python 3.12 and escher is not loading. I coded a streamlit application that will give you all the information. I am devloping a new julia based desktop application that will give you all the BIGG analyze including the metabolic modelling with in that application.

https://github.com/gauravcodepro/streamlit-BIGG-metabolic-analyzer-API

Thank you Gaurav
